### PR TITLE
Linux support for CPU

### PR DESCRIPTION
**Pull request by mizzy**

### CODE REVIEW

**Comment by kentaro** [comment]
It's more preferable that parsing `/proc/cpuinfo` without regexp because it's a simply formatted line-based file.

Please refer to the page http://stackoverflow.com/questions/11356330/getting-cpu-usage-with-golang

**Comment by mizzy** [comment]
I'll try it.

—

File: cpu/cpu_linux.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	"strings"
+	"regexp"
+)
+
+type Cpu struct{}
+
+func (self *Cpu) Collect() (result map[string]map[string]string, err error) {
+	return map[string]map[string]string{
+		"cpu": getCpuInfo(),
+	}, err
```
Comment (kentaro) [inline cpu/cpu_linux.go:15]: Set erros from `getCpuInfo()` into the `err` var.

**Comment by kentaro** [comment]
Great patch!

**Comment by mizzy** [comment]
Thanks!

—

**Pull request merged**